setup.py

```diff
@@ -22,7 +22,7 @@
     description="Replacement for the --resultlog option, focused in simplicity and extensibility",
     long_description=readme,
     keywords="pytest",
-    extras_require={"dev": ["pre-commit", "tox"]},
+    extras_require={"dev": ["pre-commit", "tox"], "testing": ["pytest-xdist"]},
```

Comment:
This is not really needed right?

Reply:
It is not :grimacing: , do you want me to remove it? It is just that I see that the tests are using xdist, but I can remove it

Reply:
I think we should remove it (the tests are pretty fast), you can always install `pytest-xdist` manually of course.